R/run_stress_test.R

```diff
@@ -33,7 +33,28 @@
 #'   excluded_companies.csv shall be excluded.
 #' @return NULL
 #' @export
-run_stress_test <- function(asset_type_arg,
+#' @examplesIf r2dii.climate.stress.test:::run_example()
+#' library(fs)
+#'
+#' (project_agnostic_path <- Sys.getenv("ST_DATA_PATH"))
+#' (project_specific_path <- Sys.getenv("ST_PROJECT_FOLDER"))
+#'
+#' outputs_path <- fs::path(project_specific_path, "outputs")
+#' if (fs::dir_exists(outputs_path)) fs::dir_delete(outputs_path)
```

Comment:
WARNING: This is dangerous because would delete files form the user's filespace. But the same problem we have every time we run `run_stress_test()` and overwrite the results form a previous run (which is rude and the reason why CRAN doesn't allow it).